src/api/directinput/JoystickInterfaceDirectInput.cpp

```diff
@@ -142,7 +143,9 @@ BOOL CALLBACK CJoystickInterfaceDirectInput::EnumJoysticksCallback(const DIDEVIC
     return DIENUM_CONTINUE;
   }
 
-  const std::string strName = pdidInstance->tszProductName ? pdidInstance->tszProductName : "";
+  std::string strName;
+  if (pdidInstance->tszProductName)
+    strName = KODI::PLATFORM::WINDOWS::FromW(std::wstring(pdidInstance->tszProductName));
```

Comment:
when unicode is enabled, the type is WCHAR[]. is there a #define i can use to check this?